tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -174,6 +174,36 @@ public function theUserSharesFileFolderWithUserUsingTheWebUI(
 		);
 	}
 
+	/**
+	 * @When /^the user shares (?:file|folder) "([^"]*)" with users "([^"]*)" using the webUI$/
```

Comment:
Could be simpler like this
```suggestion
	 * @When the user shares file/folder :folder with users :users using the webUI
```